pyproject.toml

```diff
@@ -21,3 +21,13 @@ before-test = "pip install --only-binary=numpy,scipy numpy scipy"
 [tool.cibuildwheel.linux]
 before-all = "yum install -y wget && {package}/tools/install_rust.sh"
 environment = 'PATH="$PATH:$HOME/.cargo/bin"'
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+ensure_newline_before_comments = true
+force_sort_within_sections = true
```

Comment:
How about using `profile = black`?
https://pycqa.github.io/isort/docs/configuration/profiles.html

Reply:
Yes. Thanks. The only difference is `line_length`. `force_sort_within_sections` is for pylint.

Done in dda3319